source/common/runtime/runtime_impl.cc

```diff
@@ -342,10 +342,10 @@ bool SnapshotImpl::parseEntryBooleanValue(Entry& entry) {
   absl::string_view stripped = entry.raw_string_value_;
   stripped = absl::StripAsciiWhitespace(stripped);
 
-  if (absl::EqualsIgnoreCase(stripped, "true")) {
+  if (absl::EqualsIgnoreCase(stripped, "true") || absl::EqualsIgnoreCase(stripped, "1")) {
```

Comment:
I could go either way on this, but what about trying to parse as an int, and if an init, 0 is false and anything other than 0 is true?

Reply:
That's a much better idea.